rasa_nlu/classifiers/mitie_intent_classifier.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from typing import Optional
+
+from rasa_nlu.components import Component
+from rasa_nlu.training_data import TrainingData
+
+
+class MitieIntentClassifier(Component):
+
+    name = "intent_classifier_mitie"
+
+    context_provides = {
+        "process": ["intent"],
+    }
+
+    output_provides = ["intent"]
+
+    def __init__(self, clf=None):
```

Comment:
When the component loads itself from file https://github.com/golastmile/rasa_nlu/blob/pipeline/rasa_nlu/classifiers/mitie_intent_classifier.py#L53